### ISSUE TITLE
setup scripts fail without permission

### ISSUE THREAD

**Issue by atl-mk** [mk-atlassian/slack-beeegmojis#49]
```
~ npx slack-beeegmojis -b
Figuring out where Slack is installed
Closing Slack
I like 'em BEEEEEG
https://youtu.be/WJ1I-z0pBU0
Backing up Slack
EACCES: permission denied, copyfile '/usr/lib/slack/resources/app.asar' -> '/usr/lib/slack/resources/app.asar.bak'

~ sudo npx slack-beeegmojis -b
internal/fs/utils.js:269
    throw err;
    ^

Error: EACCES: permission denied, stat '/tmp/fnm-shell-7836922'
    at Object.realpathSync (fs.js:1644:9)
    at toRealPath (internal/modules/cjs/loader.js:362:13)
    at Function.Module._findPath (internal/modules/cjs/loader.js:666:22)
    at resolveMainPath (internal/modules/run_main.js:12:25)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:65:24)
    at internal/main/run_main_module.js:17:47 {
  errno: -13,
  syscall: 'stat',
  code: 'EACCES',
  path: '/tmp/fnm-shell-7836922'
}

```

**Comment by atl-mk** [comment]
A workaround is `sudo node index.js`

**Comment by atl-mk** [comment]
Fixed in pull request in pull request #50
